do-dont/stories/index.stories.js

```diff
@@ -1,20 +1,19 @@
-import { html } from 'lit';
-import '~/do-dont/define-all';
+import '~/do-dont/define-all.js';
 
 export default {
   parameters: {
     layout: 'centered',
   },
 };
 
-const sampleButton = html`<div
+const sampleButton = /*html*/ `<div
```

Comment:
nitpick: why? lit-html here is fine, it was just for highlighting

Reply:
I wanted to be as free as possible from `lit-html` in the docs and stories (as Georges suggested) too not falsely give the impression that lit is needed for using these components.
Also, syntax highlighting in stories works with `/*html*/` as well.